Public/Src/Cache/ContentStore/Library/Extensions/StringExtensions.cs

```diff
@@ -57,5 +58,18 @@ public static string ReplaceIgnoreCase(this string str, string oldValue, string
 
             return str;
         }
+
+        /// <summary>
+        /// Returns the upper invariant when paths are case-insensitive
+        /// </summary>
+        public static string PathToUpperInvariant(this string str)

```

Comment:
As I mentioned already, I would suggest to add an extension method to AbsolutePath or PathBase instead.

Reply:
Removed!

---
In reply to: [281329075](https://github.com/Microsoft/BuildXL/pull/253#discussion_r281329075) [](ancestors = 281329075)